### PR TITLE
feat: add export minifySync() and examples

### PR DESCRIPTION
**Pull request by yisibl**
```bash
 node packages/css/examples

🚀 minifySync Time: 0.215ms
47 bytes
.foo{color:#fff;margin:20px;margin-bottom:10px}

🚀 minify Time: 4.186ms
47 bytes
.foo{color:#fff;margin:20px;margin-bottom:10px}
```

### CODE REVIEW

File: packages/css/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.11",
   "description": "Super-fast alternative for postcss",
   "homepage": "https://swc.rs",
-  "main": "./index.js",
+  "main": "./lib/index.js",
```
Comment (yisibl) [inline packages/css/package.json:6]: The index.js path in the previous version is still incorrect.

File: packages/css/lib/index.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.minify = void 0;
+exports.minifySync = exports.minify = void 0;
 const binding = require("./binding");
```
Comment (kdy1) [inline packages/css/lib/index.js]: Please remove this file

File: packages/html/lib/index.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.minify = void 0;
+exports.minifySync = exports.minify = void 0;
```
Comment (kdy1) [inline packages/html/lib/index.js]: Also this one.
Can you add `tsc -d` to the package.json, too?

File: packages/css/lib/index.js (hunk 2 above)
Comment (yisibl) [reply packages/css/lib/index.js]: Remove binding.js?

File: packages/html/lib/index.js (hunk 3 above)
Comment (yisibl) [reply packages/html/lib/index.js]: Sorry, what does that mean? This line is already `tsc -d` auto-converted.

Comment (kdy1) [reply packages/html/lib/index.js]: Nevermind, we already have it

File: packages/css/lib/index.js (hunk 2 above)
Comment (kdy1) [reply packages/css/lib/index.js]: No, `index.js`

Comment (yisibl) [reply packages/css/lib/index.js]: Does the current CI automatically generate index.js when npm publish?

Comment (kdy1) [reply packages/css/lib/index.js]: Yes

Comment (yisibl) [reply packages/css/lib/index.js]: Done

**Review by kdy1 — APPROVED** [review]
Thanks!

**Comment by yisibl** [comment]
@kdy1 v0.0.12 still missing `./lib/index.js`.

```bash
ls node_modules/@swc/css/lib
binding.d.ts binding.js   index.ts
```

**Comment by kdy1** [comment]
Should be fixed by https://github.com/swc-project/bindings/commit/c60c0f536d595791d3bc11118ba27fe5a79f852f#diff-f978361fde7fd5ba93ccd86cd7bda5b387aa2a27e29c2c77f3650b1f53424b6fR525-R527

**Comment by yisibl** [comment]
Thanks!

**Pull request merged**